src/http-api/index.js

```diff
@@ -0,0 +1,37 @@
+var Hapi = require('hapi')
+var IPFS = require('../ipfs-core')
+var debug = require('debug')
+var log = debug('api')
+
+exports = module.exports
+
+exports.start = function (callback) {
+  // start IPFS and exports.ipfs = new IPFS()
+
+  exports.ipfs = new IPFS()
```

Comment:
Kind of, but why not just pass these resources through explicitly, this could easily break, or be miss used by someone not understanding these exports